### ISSUE TITLE
There is no way to access ECS client endpoint from non-aliyun VM

### ISSUE THREAD

**Issue by jia-jerry** [denverdino/aliyungo#435]
In my office computer, when I create a VPC client and try to fetch VPC, it will fail. Because it can't access endpoint (100.100.80.75). I'm using Shanghai region.
----
```
vpc = NewVPCClientWithSecurityToken4RegionalDomain("mykey", "mysecret", "", "cn-shanghai")
```

I need a latest version of CCM(https://github.com/kubernetes/cloud-provider-alibaba-cloud). 

It is caused by this https://github.com/denverdino/aliyungo/pull/396

**Comment by denverdino** [comment]
Fixed in #436

**Comment by jia-jerry** [comment]
thx!
